### ISSUE TITLE
ADF radio blinks at higher intensity than the radiostack

### ISSUE THREAD

**Issue by gilbertohasnofb** [c172p-team/c172p#1149]
To reproduce this, first make sure the BAT switch it set to off and then set the RADIO LT to a low value like 20%. Now flick the BAT switch on the ADF radio blinks fully bright for a very short period before going back to normal. Its buttons also only lit up after a short delay, and the DME radio also has a delay for its digits. When turning the BAT off, there is also a delay on the ADF radio lights. Flick the BAT on and off to observe it several times. So there are three issues:

* ADF radio blinks at full brightness for a short period
* ADF radio button backlight is delayed both when turning on and off
* DME radio readouts are delayed both when turning on and off

**Comment by gilbertohasnofb** [comment]
@wlbragg can you reproduce this?

**Comment by wlbragg** [comment]
Yeah, this "might" be difficult to fix. First guess is the logic flow is the cause and that can be a huge can of worms to fix.

**Comment by gilbertohasnofb** [comment]
I might take a look at this if I get the chance, but it might indeed be difficult to tackle. Let's see, not the end of the world anyway.

**Comment by wlbragg** [comment]
@gilbertohasnofb 
>* ADF radio blinks at full brightness for a short period
* ADF radio button backlight is delayed both when turning on and off
* DME radio readouts are delayed both when turning on and off

I think we can add 

* ADF radio button backlight is delayed both when turning on and off
* DME radio readouts are delayed both when turning on and off
* AP radio readouts are delayed both when turning on and off

**Comment by gilbertohasnofb** [comment]
Sure, I added the last one (the other two weer already there)
